frontend/src/index.css

```diff
@@ -149,6 +172,13 @@ Other button classes are defined further down together with other classes for th
   background-color: white;
 }
 
+@media (max-height: 700px) and (orientation: landscape) {
+  .footer {
+    position: relative;
+    z-index: 0;
```

Comment:
don't think the z-index is needed here

Reply:
It is needed, as otherwise the date picker is semi visible on small screens